pkgs/applications/office/gnucash/default.nix

```
@@ -1,106 +1,210 @@
-{ fetchurl, lib, stdenv, pkg-config, makeWrapper, cmake, gtest
-, boost, icu, libxml2, libxslt, gettext, swig, isocodes, gtk3, glibcLocales
-, webkitgtk, dconf, hicolor-icon-theme, libofx, aqbanking, gwenhywfar, libdbi
-, libdbiDrivers, guile, perl, perlPackages
+{
+  # nix "native" dependencies
+  fetchurl
+, lib
+, stdenv
+
+, glibcLocales  # necessary to pass locale-related GnuCash tests
+, wrapGAppsHook # nix sugar for Gnome apps
+
+, makeWrapper   # used to wrap gnucash with Gnome sugar and the cli utils which need Perl
+
+, cmake         # tell nix to use cmake rather than make
+
+  # GnuCash requirements
```

```suggestion
{ lib
, fetchurl
, stdenv 
, glibcLocales
, wrapGAppsHook
, makeWrapper
, cmake
```

Everyone knows what eg wrapGAppsHook is doing and using the cmake build system also does not require a comment. Also the comments are way more useful at the places where the input is actually used because inputs are just a list of everything used and order does not matter for nix.

---

fixed in 072900f31f1598aca35feb66e4c553bbe69a9509